src/test/resources/features/US61_UserInfo_SK.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Librarian manage user info table
+
+  US61 As a user, I want to change the number of rows to see the user info
+
+  Background: User is already in the log in page
+    Given the user logged in as "librarian"
+
+
+  Scenario: Change number of rows in User page
+
+    And I click on "User" link
```

Comment:
user

Reply:
According to the userStory "User" being used, I just simply followed

Reply:
public  void clickOption(String option){
        String locator="//span[.='"+option+"']";
        BrowserUtils.sleep(1);
        Driver.getDriver().findElement(By.xpath(locator)).click();
    }
    
    You can implement this method
    
    And then pass in your  step defs this method ---> 
    
      @When("I click on {string} link")
    public void i_click_on_link(String string) {
       usersPage.clickOption(string); 
    }
    
    Make object from UserPage

Reply:
Hi Tijana, can you explain more why do I have to implement as a String option? 
in my US61_UserPage I locate the webElement with I use as "userLink"